sechub-schedule/src/test/java/com/daimler/sechub/domain/schedule/strategy/SchedulerStrategyIdTest.java

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.domain.schedule.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import org.junit.jupiter.api.Test;
+
+class SchedulerStrategyIdTest {
+
+    @Test
+    void test_value_to_enum_mapping_succesful() {
+        /* prepare */
+        SchedulerStrategyId fifoServe = SchedulerStrategyId.getId("first-come-first-serve");
```

Comment:
hmm... this is the execute isn't it? so not prepare....
(we test if getId method works as expected) so getIt = execution

Reply:
done